.gitignore

```diff
@@ -142,3 +142,6 @@ dmypy.json
 
 # PyCharm .idea files
 .idea/
+
+# shell scripts
+*.sh
```

Comment:
can you please remove this from the PR as it is not relevant for the scope of this PR 🙏 ?